### PR TITLE
Computing RID based on /etc/os-release instead of dotnet --info

### PR DESCRIPTION
**Pull request by rajkumar42**
Tested it with docker containers!

### CODE REVIEW

**Comment by rajkumar42** [comment]
@gregg-miskelly @caslan @chuckries @wesrupert

**Comment by chuckries** [comment]
:+1: please squash.

File: scripts/GetClrDbg.sh [hunk 1]
```diff
@@ -49,32 +48,63 @@ print_help()
...
+            __RuntimeID=rhel.7-x64
+            ;;
+        debian)
+            __RuntimeID=debian.8-x64
+            ;;
+        ol)
+            __RuntimeID=centos.7-x64
+            ;;
+        elementaryOS)
+            if [[ "$PlatformVersionID" == 0.3* ]]; then
```
Comment (gregg-miskelly) [inline scripts/GetClrDbg.sh]: >. [](start = 43, length = 1)

Does the '.' need to be escaped?

Comment (gregg-miskelly) [inline scripts/GetClrDbg.sh]: Looks like it doesn't need to be.

---
In reply to: [77927758](https://github.com/Microsoft/MIEngine/pull/437#discussion_r77927758) [](ancestors = 77927758)

**Comment by gregg-miskelly** [comment]
:shipit:

**Pull request merged**